kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -12,6 +12,14 @@ data:
     # Any errors related to config map settings can be found in the KubeMonAgentEvents table in the Log Analytics workspace that the cluster is sending data to.
 
     [log_collection_settings]
+      #  This feature in private preview and not ready for production use,please reach out us to try out this feature.
+      #  [log_collection_settings.multi_tenancy]
+      #    # High log scale MUST be enabled to use this feature. Refer to https://aka.ms/cihsmode for more details on high log scale mode
+      #    enabled = true
+      #    # Below advanced settings MUST be provided when ama-logs-multitenancy service deployed through AKS Monitoring Addon
+      #    namespaces = ["app-team-1","app-team-2"]
+      #    namespace_settings = ["app-team-1:throttle_rate=2000","app-team-2:throttle_rate=5000"]
```

Comment:
why only throttle rate is exposed for config?
  "storage_type" => "filesystem",
  "mem_buf_limit" => "10m",
  "buffer_chunk_size" => "1m",
  "buffer_max_size" => "5m",
  "throttle_rate" => "1000",
  "throttle_window" => "300",
  "throttle_interval" => "1s",
  "out_forward_worker_count" => "10",
  "out_forward_retry_limit" => "10",
  "out_forward_storage_total_limit_size" => "2G",
  "out_forward_require_ack_response" => "false",
  "disable_throttle" => "false"

Reply:
all these are configurable through config map. we dont expect customer to configure other than throttle_rate. For others, we can suggest as needed and having all these in the configamp can be error prone.